src/Akka.Persistence.Hosting/JournalOptions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="JournalOptions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.Hosting
+{
+    public abstract class JournalOptions
```

Comment:
Needs XML-DOC

Reply:
I'd still add this for people who want to add Hosting support to Akka.Persistence plugins - what are some things you need to do to implement this class appropriately / correctly? Doesn't need to be super-detailed, but broad strokes would be good.